src/ui/interaction/container/scanner.tsx

```diff
@@ -56,19 +49,22 @@ export const ScannerContainer = (props: Props) => {
       })
     }
 
-    requestCameraPermission().then(() => {
-      setTimeout(() => setCameraReady(true), 200)
```

Comment:
will bring `setCameraReady` back. But is there any stutter now, without the delay? It feels to me to be fine, and I'm trying to avoid making the user wait even longer than the usual... an extra 200ms is quite noticeable

Reply:
ok so we don't really need it, and I closed #1567 and will include a different fix

Reply:
I think we can just do this actually: https://github.com/moaazsidat/react-native-qrcode-scanner#reactivate-1
it was `true` for some reason.....

Reply:
it shouldn't be false by default, since in case of an unsuccessful scan, it will not reactivate (i think). It has a 3s timeout when unsuccessful.

Reply:
I think this is moot now